src/test/kotlin/no/nav/tiltaksarrangor/IntegrationTest.kt

```diff
@@ -116,3 +116,12 @@ class IntegrationTest {
 		).serialize()
 	}
 }
+
+private fun getKafkaImage(): String {
```

Comment:
var det noe som feilet sånn oppsettet var før, eller er det en annen grunn til å skille på prosessortype?

Reply:
Forskjellen er at testene kjører 10-20 sekunder raskere, for oss med M1/M2 Mac, med et image bygget for arm prosessorer.